### PR TITLE
New Feature to allow filtering the popular tags module by a Parent Tag.

### PR DESCRIPTION
**Pull request by tonypartridge**
Pull Request for Issue #17466  .

### Summary of Changes
Added a Parent filter Tag to the Popular Tags Module

### Testing Instructions
Apply Patch, select a parent tag or tags in the Popular Tags Module, results should only include the child tags.

### CODE REVIEW

**Comment by brianteeman** [comment]
Fixed some codestlye issues with tabs instead of spaces

**Comment by tonypartridge** [comment]
Thanks @brianteeman.

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on a4cfa88a71399f0f6011a2fce44e1c132f87d53b<br /><br />Thanks @tonypartridge <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17562">issues.joomla.org/tracker/joomla-cms/17562</a>.</sub>

**Comment by ghost** [comment]
Set "Easy Test" to "Yes" in Issue Tracker removes here "New Feature". @brianteeman can you please reassign Label?

File: language/en-GB/en-GB.mod_tags_popular.ini [hunk 1]
```diff
@@ -31,4 +31,6 @@ MOD_TAGS_POPULAR_FIELDSET_CLOUD_LABEL="Cloud Layout"
 MOD_TAGS_POPULAR_MAX_DESC="Sets the maximum number of tags to display in the module."
 MOD_TAGS_POPULAR_MAX_LABEL="Maximum Tags"
 MOD_TAGS_POPULAR_NO_ITEMS_FOUND="No Tags found."
+MOD_TAGS_POPULAR_PARENT_TAG_LBL="Parent Tag"
```
Comment (Quy) [inline language/en-GB/en-GB.mod_tags_popular.ini]: Change `LBL` to `LABEL` to be consistent.

Comment (tonypartridge) [reply language/en-GB/en-GB.mod_tags_popular.ini]: Seriously? I thought we are moving everything to LBL? God I'll never keep up on this lol.

Comment (Quy) [reply language/en-GB/en-GB.mod_tags_popular.ini]: I don't know either. I'm just going by the other labels.

**Comment by brianteeman** [comment]
I wasn't aware of any change to USE LBL

**Comment by tonypartridge** [comment]
@brianteeman I'm tried. Cleary I was wrong looking at the CMS Language File Labeling.

File: language/en-GB/en-GB.mod_tags_popular.ini [hunk 2]
```diff
@@ -31,4 +31,6 @@ MOD_TAGS_POPULAR_FIELDSET_CLOUD_LABEL="Cloud Layout"
 MOD_TAGS_POPULAR_MAX_DESC="Sets the maximum number of tags to display in the module."
 MOD_TAGS_POPULAR_MAX_LABEL="Maximum Tags"
 MOD_TAGS_POPULAR_NO_ITEMS_FOUND="No Tags found."
+MOD_TAGS_POPULAR_PARENT_TAG_LABEL="Parent Tag"
```
Comment (Quy) [inline language/en-GB/en-GB.mod_tags_popular.ini]: Alpha order. Swap lines. Sorry I missed it last time.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on bd1a52e648d8a9ad765a10e15ffb40b43cc099d4<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17562">issues.joomla.org/tracker/joomla-cms/17562</a>.</sub>

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on bd1a52e648d8a9ad765a10e15ffb40b43cc099d4<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17562">issues.joomla.org/tracker/joomla-cms/17562</a>.</sub>

**Comment by ghost** [comment]
RTC after two successful tests.

**Comment by ghost** [comment]
Thanks for Coding and Tests.

**Pull request merged**